serveradmin/jinja2.py

```diff
@@ -1,17 +0,0 @@
-"""Serveradmin
-
-Copyright (c) 2018 InnoGames GmbH
-"""
-
-from jinja2 import Environment
-from django.contrib.staticfiles.storage import staticfiles_storage
-from django.core.urlresolvers import reverse
-
-
-def environment(**options):
-    env = Environment(**options)
-    env.globals.update({
-        'static': staticfiles_storage.url,
-        'url': reverse,
-    })
-    return env
```

Comment:
`jinja2` was/is used by `ignetworking`.

Reply:
This jinja2.py was copied straight from the django docs. It makes some more django env variables available in the templates like static and url. LB pushing doesn't use those though, I diffed the the config with and without this patch and there is no difference.